src/DssSpell.sol

```diff
@@ -59,187 +51,230 @@ contract DssSpellAction is DssAction {
+    address internal immutable D3M_HUB        = DssExecLib.getChangelogAddress("DIRECT_HUB");
+    address internal immutable D3M_MOM        = DssExecLib.getChangelogAddress("DIRECT_MOM");
+    address internal immutable MCD_PSM_USDC_A = DssExecLib.getChangelogAddress("MCD_PSM_USDC_A");
+    address internal immutable SUSDS          = DssExecLib.getChangelogAddress("SUSDS");
 
-                // Init new Flapper with want: 0.98 * WAD
-                want: 98 * WAD / 100,
 
-                // Init new Flapper with pip: SWAP_ONLY_FLAP_SKY_ORACLE (0x61A12E5b1d5E9CC1302a32f0df1B5451DE6AE437)
-                pip: SWAP_ONLY_FLAP_SKY_ORACLE,
+    address internal constant DIRECT_SPK_AAVE_LIDO_USDS_PLAN     = address(0); // TODO: replace
+    address internal constant DIRECT_SPK_AAVE_LIDO_USDS_POOL     = address(0); // TODO: replace
+    address internal constant DIRECT_SPK_AAVE_LIDO_USDS_ORACLE   = address(0); // TODO: replace
+    address internal constant DIRECT_SPK_AAVE_LIDO_USDS_OPERATOR = address(0); // TODO: replace
 
-                // Init new Flapper with pair: PAIR_USDS_SKY (0x2621CC0B3F3c079c1Db0E80794AA24976F0b9e3c)
-                pair: UNIV2USDSSKY,
-
-                // Init new Flapper with usds: dss.chainlog.getAddress("USDS")
-                usds: USDS,
-
-                // Init new Flapper with splitter: dss.chainlog.getAddress("MCD_SPLIT")
-                splitter: MCD_SPLIT,
-
-                // Init new Flapper with prevChainlogKey: bytes32(0)
-                prevChainlogKey: bytes32(0),
-
-                // Init new Flapper with chainlogKey: "MCD_FLAP"
-                chainlogKey: "MCD_FLAP"
-            })
-        );
-
-        // Init new OracleWrapper by calling FlapperInit.initOracleWrapper with the following parameters:
-        FlapperInit.initOracleWrapper(
-            // Note: DssInstance is required by the init library
-            dss,
-
-            // Init new OracleWrapper with wrapper_: SWAP_ONLY_FLAP_SKY_ORACLE (0x61A12E5b1d5E9CC1302a32f0df1B5451DE6AE437)
-            SWAP_ONLY_FLAP_SKY_ORACLE,
-
-            // Init new OracleWrapper with divisor: 24,000
-            24_000,
-
-            // Init new OracleWrapper with clKey: "FLAP_SKY_ORACLE"
-            "FLAP_SKY_ORACLE"
-        );
-
-        // Additional Actions
+    function actions() public override {
+        // ---------- Stability Scope Parameter Changes  ----------
+        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-16-sfs-ssr-dsr-spark-effective-dai-borrow-rate-changes/25257
 
-        // Increase vow.hop by 1386 seconds from 10249 seconds to 11635 seconds
-        DssExecLib.setValue(MCD_SPLIT, "hop", 11_635);
+        // Stability Fee (SF) changes:
+        // Note: only heading, changes follow
 
-        // Decrease vow.bump by 40000 USDS from 65000 USDS to 25000 USDS
-        DssExecLib.setValue(MCD_VOW, "bump", 25_000 * RAD);
+        // WBTC-A: Increase by 1.5 percentage points, from 7.75% to 9.25%
```

Comment:
nit, There is 2 spaces after `:` in exec sheet but the extra one is removed here. IMHO, this can cause confusion when reviewers/other people compare items 1-1 from exec sheet. As checklist outlines to [`copy`](https://github.com/makerdao/pe-checklists/blob/5208017c2c5e04f9ec03817856e06ff9d0fe9ad1/spell/spell-reviewer-mainnet-checklist.md?plain=1#L21) the instruction.

Reply:
it is now fixed in the exec sheet!